packages/lesswrong/components/ea-forum/EABestOfPage.tsx

```diff
@@ -63,184 +48,66 @@ const styles = (theme: ThemeType): JssStyles => ({
-    marginRight: 8,
-  },
-  postListItemTitle: {
-    fontSize: 18,
-    marginBottom: 8,
-    lineHeight: "25px",
-    overflow: "hidden",
-    display: "-webkit-box",
-    "-webkit-box-orient": "vertical",
-    "-webkit-line-clamp": 2,
-  },
-  postListItemMeta: {
-    display: "flex",
-    marginBottom: 8,
-    fontSize: 14,
-    lineHeight: "20px",
-    color: theme.palette.grey[600],
-  },
-  commentCount: {
-    minWidth: 58,
-    marginLeft: 4,
-    display: "flex",
-    alignItems: "center",
-    cursor: "pointer",
-    "& svg": {
-      height: 18,
-      marginRight: 1,
-    },
-    "&:hover": {
-      color: theme.palette.grey[800],
-      opacity: 1,
-    },
-  },
-  postListItemPreview: {
-    fontSize: 14,
-    lineHeight: "20px",
-    color: theme.palette.grey[600],
-    position: "relative",
-    overflow: "hidden",
-    display: "-webkit-box",
-    "-webkit-box-orient": "vertical",
-    "-webkit-line-clamp": 3,
-    marginTop: "auto",
-    marginBottom: "auto",
-  },
-  postListItemImage: {
-    height: 140,
-    maxWidth: 150,
-    objectFit: "cover",
-    borderRadius: theme.borderRadius.default,
-    [theme.breakpoints.down("xs")]: {
-      display: "none",
-    },
-  },
-  // Sequence or collection card
-  sequenceCard: {
-    borderRadius: theme.borderRadius.default,
-    background: theme.palette.panelBackground.default,
-    fontFamily: theme.palette.fonts.sansSerifStack,
-    fontWeight: 500,
-    marginLeft: 8, // Account for box shadow
-    boxShadow: `/* The top layer shadow */
-      -1px 1px 10px ${theme.palette.greyAlpha(0.15)},
-      /* The second layer */
-      -8px 8px 0 0px ${theme.palette.panelBackground.default},
-      /* The second layer shadow */
-      -10px 10px 10px -1px ${theme.palette.greyAlpha(0.15)}`,
-  },
-  sequenceCardImageWrapper: {
-    position: "relative",
-  },
-  sequenceCardImage: {
-    width: "100%",
-    height: SEQUENCE_CARD_IMAGE_HEIGHT,
-    objectFit: "cover",
-    borderTopLeftRadius: theme.borderRadius.default,
-    borderTopRightRadius: theme.borderRadius.default,
-    zIndex: Z_IMAGE,
-  },
-  sequenceReadProgress: {
-    position: "absolute",
-    zIndex: Z_OVERLAY,
-    fontSize: 12,
-    top: 8,
-    left: 8,
-    borderRadius: 14,
-    backgroundColor: theme.palette.panelBackground.translucent3,
-    padding: "6px 8px",
-  },
-  sequenceCardText: {
-    padding: 16,
-  },
-  sequenceCardTitle: {
-    fontSize: 18,
-    lineHeight: "20px",
-    overflow: "hidden",
-    display: "-webkit-box",
-    "-webkit-box-orient": "vertical",
-    "-webkit-line-clamp": 2,
-    marginBottom: 6,
-  },
-  sequenceCardMeta: {
-    fontSize: 14,
-    lineHeight: "20px",
+    fontSize: 16,
+    fontWeight: 600,
     color: theme.palette.grey[600],
   },
-  // Featured audio card
-  audioCard: {
-    marginTop: 16,
+  postsItem: {
+    maxWidth: "unset",
   },
 });
 
+const featuredCollectionsCollectionIds = [
+  "MobebwWs2o86cS9Rd", // EA Handbook
```

Comment:
It would be nice to make it easier to change all these, but seeing as we'll be editing this very rarely leaving these hardcoded for now seems ok